airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -48,13 +68,26 @@ def __init__(
         aws_conn_id: str = 'aws_default',
         max_messages: int = 5,
         wait_time_seconds: int = 1,
+        visibility_timeout: Optional[int] = None,
+        message_filtering: Optional[str] = None,
+        message_filtering_match_values: Optional[Any] = None,
+        message_filtering_config: Optional[Any] = None,
```

Comment:
`Optional[Any]` is just `Any`, so the Optional part is useless

Reply:
Noted, thank you. Fixed